.npmrc

```diff
@@ -0,0 +1 @@
+registry = https://registry.npmjs.org/
```

Comment:
this is by default, is the intent here to force the default repository and ignore the local one?

Reply:
yes, since many of us have a local .npmrc with corp artifactory, this will prevent someone from accidentally committing the package-lock.json with corp artifactory

should save time debugging for devs